test-requirements.txt

```diff
@@ -5,6 +5,7 @@
 mock
 ansible
 hacking<0.12,>=0.11.0 # Apache-2.0
+netifaces
```

Comment:
ok, tox considering both two files as dependeny. so  any one is enough,  as per your comment i will move it to requirements.txt